<launch>

  <!-- This is the format for a comment in HTML --> 


  <!-- Launches the geonav_transform file --> 
  <include file="$(find geonav_transform)/launch/geonav_transform.launch" >
  </include>

  <!-- creates all the parameters --> 
  <param name="/Kp_xy_G" value="0.7" />
  <param name="/Kp_psi_G" value="1.0" />
  <param name="/Kd_xy_G" value="2.0" />
  <param name="/Kd_psi_G" value="1.0" />
  <param name="/Ki_xy_G" value="0.0" />
  <param name="/Ki_psi_G" value="0.0" />

  <!-- Launches codes --> 
  <node pkg="amore" type="gps_imu" name="Pose_Converter" output="screen">
  </node>
  <node pkg="amore" type="Station_keeping" name="SK_Controller" output="screen">
  </node>

</launch>
